server/lib/nl/detection/place.py

```diff
@@ -275,6 +276,16 @@ def get_place_from_dcids(place_dcids: List[str], debug_logs: Dict) -> any:
                              place_type=ptype,
                              country=country)
 
+  # If there were detected places that there wasn't any place info for, assume
```

Comment:
Should we return entities separate from places?

Instead of `Place` type, it can be an `Entity` type?